hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/AbstractOmBucketReadWriteOps.java

```diff
@@ -46,10 +47,12 @@ public abstract class AbstractOmBucketReadWriteOps extends BaseFreonGenerator
       LoggerFactory.getLogger(AbstractOmBucketReadWriteOps.class);
 
   @Option(names = {"-g", "--size"},
-      description = "Generated data size (in bytes) of each key/file to be " +
-          "written.",
-      defaultValue = "256")
-  private long sizeInBytes;
+      description = "Generated data size of each key/file to be written. " +
+          "You can specify the size using data units like 'GB', 'MB', 'KB', " +
```

Comment:
We can add a description to indicate that `size` is base 2 binary.

Reply:
Added this point to the description, thank you!